packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -86,6 +92,32 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
     [onMouseUp],
   );
 
+  const handleFocus = React.useCallback(
+    (ev: FocusEvent) => {
+      ev.preventDefault();
+      ev.stopPropagation();
+
+      const document = iframeRef.current?.contentDocument;
+      const fiberNav = FiberNavigator.fromDOMNode(ev.target);
+      const parent = fiberNavFindOwnerInJSONTree(fiberNav, jsonTree);
+
+      document?.body.focus();
```

Comment:
why do we need to focus body? we should just detect if the focus is on body

Reply:
You are right. I've added it before because the focus was always in the iframe - on the previous method that I was trying to do. So this is some code I forgot to delete. Thx!